lib/blinkist-airbrake-scrubber/airbrake.rb

```diff
@@ -0,0 +1,6 @@
+# Prepend the original Airbrake module
+module Airbrake
+  class << self
+    prepend Blinkist::Airbrake::Scrubber
+  end
+end
```

Comment:
I don't like that this lives in a separate file now. There are two problems with that:
1. Is is now hidden, i.e. not in a place where you would immediately look (the main file of the gem)
2. It is load-order dependant. I.e. if you load this file before `Scrubber`, everything breaks.

Reply:
Good point, I will revert this